src/test/groovy/io/seqera/wave/util/RegHelperTest.groovy

```diff
@@ -284,4 +283,23 @@ class RegHelperTest extends Specification {
                 .hash()
                 .toString()
     }
+
+    @Unroll
+    def 'should validate the provided custom container image name'(){
+        expect:
+        RegHelper.isValidImageName( IMAGENAME ) == ISVALID
+
+        where:
+        IMAGENAME               | ISVALID
+        'foo'                   | true
+        'foo:1.1'               | true
```

Comment:
The colon symbol should not be allowed, right? 

This is because when we provide an `imageName`, the image tag is still defined by Wave (see e.g. lines 197-204 in your edits to ContainerTokenController).

Reply:
done